extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -429,6 +429,15 @@ export async function makeHttpRequest(account: azdata.Account, subscription: azu
 	return result;
 }
 
+export async function getManagedDatabases(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, managedInstance: azureResource.AzureGraphResource, ignoreErrors: boolean): Promise<GetManagedDatabasesResult> {
```

Comment:
I am trying to match the type of managed instance api

Reply:
What API is this?

It's possible that whoever first implemented that didn't type it correctly. If so then it would be preferable to take the time now to do that (even if it just extends AzureGraphResource and nothing else. It's more about ensuring that users are calling this with an actual managed instance instead of any random resource)